doc/content/devices/models/mkr1310/temp.md

```diff
@@ -0,0 +1,205 @@
+  if (!connected) {
+    Serial.println("Something went wrong; are you indoor? Move near a window and retry");
+    while (1) {}
+  }
+}
+```
+
+In the `void loop()` we initialize the variables containing the values of Temperature and Humidity. We multiply these values by 100, since we can not send decimal numbers directly over LoRaWAN. Using the `byte payload` function we contain all the values we need for sending. Then using `mode.write` we finally send the package off to the network, ready to be received by {{% tts %}}.
+
+```cpp
+void loop() {
+  int t = dht.readTemperature() * 100;
+  int h = dht.readHumidity() * 100;
+
+  byte payload[4];
+  payload[0] = highByte(t);
+  payload[1] = lowByte(t);
+  payload[2] = highByte(h);
+  payload[3] = lowByte(h);
+
+  Serial.println("Temperature: ");
+  Serial.println(dht.readTemperature());
+  Serial.println("Humidity: ");
+  Serial.println(dht.readHumidity());
+
+  modem.beginPacket();
+  modem.write(payload, sizeof(payload));
+  modem.endPacket(true);
+  delay(100000);
+}
+```
+
+### Entire code
+
+```cpp
+#include <MKRWAN.h>
+#include <DHT.h>
+
+LoRaModem modem(Serial1);
+
+String appEui = "";
+String appKey = "";
+
+DHT dht(6, DHT11);
+
+void setup() {
+  // put your setup code here, to run once:
+  Serial.begin(115200);
+  while (!Serial);
+  // change this to your regional band (eg. US915, AS923, ...)
+  if (!modem.begin(EU868)) {
+    Serial.println("Failed to start module");
+    while (1) {}
+  };
+  
+  dht.begin();
+
+  int connected = modem.joinOTAA(appEui, appKey);
+  if (!connected) {
+    Serial.println("Something went wrong; are you indoor? Move near a window and retry");
+    while (1) {}
+  }
+}
+
+void loop() {
+  int t = dht.readTemperature() * 100;
+  int h = dht.readHumidity() * 100;
+
+  byte payload[4];
+  payload[0] = highByte(t);
+  payload[1] = lowByte(t);
+  payload[2] = highByte(h);
+  payload[3] = lowByte(h);
+
+  Serial.println("Temperature: ");
+  Serial.println(dht.readTemperature());
+  Serial.println("Humidity: ");
+  Serial.println(dht.readHumidity());
+
+  modem.beginPacket();
+  modem.write(payload, sizeof(payload));
+  modem.endPacket(true);
+  delay(100000);
+}
+```
+## Uploading the code
+
+Now to upload the code to the MKR WAN 1310. First plug the Arduino into your computer. In the IDE, click on **Select Board** at the top, and select your **Arduino MKR WAN 1310**.
+
+{{< figure src="../select-board.png" alt="Selecting Board" >}}
+
+Once that is done you can upload the file to the board (Using the **->** button). Once the code is uploaded successfully, open the Serial Monitor. You can open it by clicking the **icon in the top-right corner**.
+
+{{< figure src="../serial-monitor.png" alt="Serial Monitor button highlighted" >}}
+
+Here you can check if the temperature and humidity values are showing up correctly.
+
+{{< figure src="../serial-monitor-temp.png" alt="Serial Monitor" width="55%" >}}
+
+## {{% tts %}}
```

Comment:
Please don't use this as a title, as it literally doesn't explain anything. It would be weird when doing docs search too. You can put something like "Decode temperature and humidity data" or something like that